marklogic-data-hub-central/ui-custom/src/components/ImageGallery/ImageGallery.scss

```diff
@@ -0,0 +1,20 @@
+.ImageGallery {
+    display: flex;
+    justify-content: center;
+    .carouselContainer {
+        width: 150px;
```

Comment:
@iveyMarklogic, how does one change the size of the image carousel? For example, what if I wanted to make it twice as big? It would be good to allow this via config properties, how can we do this?

When I adjust this CSS, I don't see the size change.